plugins/org.eclipse.osee.orcs.rest/src/org/eclipse/osee/orcs/rest/internal/TransactionEndpointImpl.java

```diff
@@ -155,8 +152,8 @@ public Response purgeTxs(String txIds) {
    @Override
    public Response purgeUnusedBackingDataAndTransactions() {
       orcsApi.userService().requireRole(CoreUserGroups.OseeAccessAdmin);
-      orcsApi.getTransactionFactory().purgeUnusedBackingDataAndTransactions();
-      return Response.ok().build();
+      int[] rtn = orcsApi.getTransactionFactory().purgeUnusedBackingDataAndTransactions();
```

Comment:
Can we inline this or handle potential errors here?

Reply:
Previously this was returning this kind of array which is somewhat useful.  Perhaps I could update it to return a pojo with more details about what was purged when I get around to updating to purge unused transactions.